packages/sinuous/src/index.d.ts

```diff
@@ -92,10 +94,10 @@ declare namespace sinuous {
 
   /**
    * Creates a new hyperscript function with the passed reactive API.
-   * @param options
-   * @param isSvg
+   * @param apiOverloads Written into the API object
+   * @param state For inflight changes; `hs` is fixed to `{ svgMode: true }`
```

Comment:
I'm kinda implying in this sentence that state could be used for more than this, which may not be true if the project is stabilizing. I just needed an object reference.